lib/streamlit/elements/arrow.py

```diff
@@ -119,6 +280,30 @@ def dataframe(
 
             To configure the index column(s), use ``_index`` as the column name.
 
+        key : str
+            An optional string to use as the unique key for this element when used in combination
+            with ```on_select```. If this is omitted, a key will be generated for the widget based
+            on its content. Multiple widgets of the same type may not share the same key.
+
+        on_select : "ignore" or "rerun" or callable
+            Controls the behavior in response to selection events on the table. Can be one of:
+
+            - "ignore" (default): Streamlit will not react to any selection events in the chart.
+            - "rerun": Streamlit will rerun the app when the user selects rows or columns in the table.
+              In this case, ```st.dataframe``` will return the selection data as a dictionary.
+            - callable: If a callable is provided, Streamlit will rerun and execute the callable as a
```

Comment:
nit: maybe worth to mention if the callback accepts any parameters (probably not right?)

Reply:
Nope, doesn't accept any parameters. But thats probably something for Debbie to unify since it uses the same text as other comparable instances.